packs/_source/classfeatures/paladin/paladin-features/lay-on-hands.json

```diff
@@ -50,13 +50,13 @@
               "target": "",
               "value": "1",
               "scaling": {
-                "mode": "",
+                "mode": "amount",
                 "formula": ""
               }
             }
           ],
           "scaling": {
-            "allowed": false,
+            "allowed": true,
             "max": ""
```

Comment:
You should set a max scaling value here so it can display the slider rather than the text input.

<img width="438" alt="Screenshot 2024-09-30 at 16 13 04" src="https://github.com/user-attachments/assets/ccbd9e67-4d66-4eee-8b9f-93eb76507de2">

```suggestion
            "max": "@item.uses.value"
```